lib/core/presentation/widgets/collaborator/spotline_event_item.dart

```diff
@@ -0,0 +1,180 @@
+import 'dart:ui';
+
+import 'package:app/core/config.dart';
+import 'package:app/core/domain/common/entities/common.dart';
+import 'package:app/core/domain/event/entities/event.dart';
+import 'package:app/core/presentation/widgets/image_placeholder_widget.dart';
+import 'package:app/core/presentation/widgets/theme_svg_icon_widget.dart';
+import 'package:app/core/utils/date_format_utils.dart';
+import 'package:app/core/utils/image_utils.dart';
+import 'package:app/gen/assets.gen.dart';
+import 'package:app/theme/color.dart';
+import 'package:app/theme/spacing.dart';
+import 'package:app/theme/typo.dart';
+import 'package:cached_network_image/cached_network_image.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_screenutil/flutter_screenutil.dart';
+
+class SpotlineEventItem extends StatelessWidget {
```

Comment:
Typo issue, should be Spotlight event, btw, these only used in collaborator, please consider put it in `pages/collaborator/widgets`